code/modules/mapfluff/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -36,7 +36,8 @@
 	return ..()
 
 /obj/structure/lavaland/ash_walker/deconstruct(disassembled)
-	new /obj/item/assembly/signaler/anomaly (get_step(loc, pick(GLOB.alldirs)))
+	var/obj/item/assembly/signaler/anomaly/core_to_drop = pick(subtypesof(/obj/item/assembly/signaler/anomaly))
```

Comment:
```suggestion
	var/core_to_drop = pick(subtypesof(/obj/item/assembly/signaler/anomaly))
```
We don't have any use of this since we're not doing anything to the anomaly but new it